### PR TITLE
Adding Cappuccino

### PR DESCRIPTION
**Pull request by Brand00wn**

### CODE REVIEW

File: src/recipes/cappuccino.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        "Much will depend on your particular machine. Turn on the steamer and let it warm up. Meanwhile, pour the milk into a stainless steel pitcher. Submerge the tip of the steam wand at least half an inch below the milk surface and turn on the steam switch. As the foam rises, gradually lower the pitcher so the tip of the wand remains just below the milk surface. As the milk begins to heat up, tilt the pitcher slightly to swirl the milk and continue to steam until the side of the pitcher becomes almost too hot to hold. Turn off the steam while still holding the tip of the wand under the milk surface.",
+        "Very slowly pour the milk into the cappuccino in one of two ways:
+
+-For a smooth cappuccino in which both the warmed and foamed milk on top is evenly integrated throughout the beverage, gently pour the milk over the espresso in a thin, steady stream.
+-For a cappuccino with a foamy head, hold back the foam with a spoon and gently pour the milk over the espresso. Spoon the foam over the top.",
```
Comment (alfg) [inline src/recipes/cappuccino.json:29]: Can these be separated into their own items? Like a) and b). This won't pass as valid json.

**Review by alfg — CHANGES_REQUESTED** [review]
Image is too large. Please resize to 600x400 200KB max in filesize, as JPG. Thank you.

File: src/recipes/cappuccino.json (hunk 1 above)
Comment (Brand00wn) [reply src/recipes/cappuccino.json:29]: Ohh, fine. I'll fix here, thx!

**Comment by alfg** [comment]
#819

**Pull request closed without merging**